### PR TITLE
refactor: create salon details in admin view

### PR DESCRIPTION
**Pull request by pawelsobkowski**
# Status

 - [x] Ready ✅

# Type

- [x] Feat
- [x] Refactor

# Related issue

Voirsy/voirsy-front-end#23

# Description
Refactoring actual admin view, add Details, Calendar and Portfolio view

### CODE REVIEW

**Comment by pawelsobkowski** [comment]
Generally, most of the view is done. IDK, if it makes any sense to wait for backend for the rest of the required features. We can merge this and create a task to finish admin view with desired features when will be available. What do you think @zawolanski ?

**Comment by zawolanski** [comment]
> Generally, most of the view is done. IDK, if it makes any sense to wait for backend for the rest of the required features. We can merge this and create a task to finish admin view with desired features when will be available. What do you think @zawolanski ?

I fully agree with you. There is no sense to wait till the backend will be finished.

File: src/components/AdminPanelNavigation/adminPanelNavigation.list.tsx [hunk 1]
```diff
@@ -1,38 +1,48 @@
...
+const SalonsNavigation = ({ data }: { data: Pick<Salon, '_id' | 'name' | 'address' | 'city' | 'type'>[] }) => {
   const [translation] = useTranslation();
+  const { salonId } = useParams<{ salonId: string }>();
 
   return (
-    <Stack spacing={2.5}>
+    <Stack spacing={2}>
       <Typography variant="h5">{translation('admin:salonTemplate.heading.mySalons')}</Typography>
+      <Button variant="outlined" color="primary" endIcon={<Add />} size="large">
+        Add salon
```
Comment (zawolanski) [inline src/components/AdminPanelNavigation/adminPanelNavigation.list.tsx]: Translation.

File: src/templates/Salons/index.tsx [hunk 2]
```diff
@@ -21,36 +28,31 @@ const SalonsTemplate = ({ children }: { children: ReactNode }) => {
...
+      <Stack direction="row" spacing={2}>
+        <CustomSalonsNavigation>
+          <SalonsNavigation data={data} />
+        </CustomSalonsNavigation>
+        <Box display="flex" flexDirection="column" flexGrow={1}>
+          {data.length > 0 ? (
+            <>
+              {/* <CustomEditHeader>
+                <Typography variant="h5">Salon</Typography>
+              </CustomEditHeader> */}
```
Comment (zawolanski) [inline src/templates/Salons/index.tsx]: Is this comment could be deleted?

**Review by zawolanski — CHANGES_REQUESTED** [review]

**Review by zawolanski — APPROVED** [review]

**Pull request merged**